x-pack/solutions/observability/test/api_integration_deployment_agnostic/apis/ai_assistant/elasticsearch_tool/elasticsearch_tool.spec.ts

```diff
@@ -0,0 +1,426 @@
+import expect from '@kbn/expect';
+import { MessageRole, type Message } from '@kbn/observability-ai-assistant-plugin/common';
+import { ELASTICSEARCH_FUNCTION_NAME } from '@kbn/observability-ai-assistant-plugin/server/functions/elasticsearch';
+import { createLlmProxy, LlmProxy } from '../utils/create_llm_proxy';
+import type { DeploymentAgnosticFtrProviderContext } from '../../../ftr_provider_context';
+
+const SYSTEM_MESSAGE = `You are an AI assistant that can help with Elasticsearch operations. You have access to the ${ELASTICSEARCH_FUNCTION_NAME} tool.`;
+
+export default function ApiTest({ getService }: DeploymentAgnosticFtrProviderContext) {
+  const log = getService('log');
+  const observabilityAIAssistantAPIClient = getService('observabilityAIAssistantApi');
+
+  const messages: Message[] = [
+    {
+      '@timestamp': new Date().toISOString(),
+      message: {
+        role: MessageRole.User,
+        content: 'Test message',
+      },
+    },
+  ];
+
+  describe('/internal/observability_ai_assistant/chat - Elasticsearch Tool Restrictions', function () {
```

Comment:
After these tests are moved to the `tool: elasticsearch` test suite, the describe block can be renamed.

```suggestion
  describe('tool restrictions', function () {
```